.prettierignore

```diff
@@ -10,4 +10,14 @@ build
 tests
 webdriver
 *.yml
-*.json
\ No newline at end of file
+*.json
+docs/web-apps/automated-testing/cucumberjs-playwright/yaml.md
```

Comment:
I'm a bit worried about the maintainability of this ignore list. Anyone who contributes to the docs and adds code snippets would need to know whether the file they are editing is being ignored or not. Even in this PR, I see one snippet that's inadvertently had prettifying applied.

Is there any interest in setting this prettier config: `embeddedLanguageFormatting: "off"`

They are just code snippets after all. If a reader copy-pastes the snippet, they can apply whatever prettifier they have configured locally.

Reply:
@mhan83 I think that's a valid option moving forward, as most of the issues are with the code blocks in general

Reply:
Great. But I think its best to revert the initial prettyfier changes and re-run with the new config; there are still lots of snippets that have been improperly formatted.

Reply:
^Agreed.